stix_shifter/stix_translation/src/modules/guardium/json/from_stix_map.json

```diff
@@ -0,0 +1,87 @@
+{
+  "x_Guardium": {
+    "fields": {
+      "datacategory": ["datacategory"],
+      "start": ["QUERY_FROM_DATE"],
+      "end": ["QUERY_TO_DATE"],
+      "remotesource": ["REMOTE_SOURCE"],
+      "service_name": ["SERVICENAME"]
+    }
+  },
+  "START": {
+    "fields": {
+      "value": ["QUERY_FROM_DATE"]
+    }
+  },
+  "STOP": {
+    "fields": {
+      "value": ["QUERY_TO_DATE"]
+    }
+  },
+  "x_Guardium_db_info": {
+    "fields": {
+      "database_name": ["DATABASENAME"],
+      "value": ["DATABASENAME"]
+    }
+  },
+  "x_Guardium_exception": {
+    "fields": {
+      "errorcode": ["ERRORCODE"]
```

Comment:
As with database_name, you could include this under the one x_guardium object unless you planned to have more fields specific to an exception object. In any case, you'd want to have the key lower-cased and use hyphens, so `x-guardium-exception`.

Reply:
done